packages/reactive_ble_mobile/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 5.4.0
+*  Use proper platform_interface dependency version #837
```

Comment:
Always add an empty line after a header in Markdown.

Here and in the other file, please.